src/devTools.js

```diff
@@ -261,9 +315,10 @@ class DevToolsEnhancer {
   }
 
   enhance = (options = {}) => {
+    const instanceId = generateId(options.instanceId);
     this.init({
       ...options,
-      hostname: getHostForRN(options.hostname || 'localhost')
+      hostname: options.hostname,
```

Comment:
this is overwritten when resolving a promise anyway(the proper value would be in `socketOptions`)

Reply:
Makes sense... feel free to keep this change, despite my overall request for simplicity